data/110/880/597/9/1108805979.geojson

```diff
@@ -17,6 +17,12 @@
     "label:eng_x_preferred_placetype":[
         "state"
     ],
+    "label:hin_x_preferred_longname":[
+        "\u0924\u0947\u0932\u0902\u0917\u093e\u0928\u093e \u0915\u093e \u0930\u093e\u091c\u094d\u092f"
```

Comment:
Please leave off the placetype from the preferred longname for Hindi in India (all regions). This value could be moved to an alt label? Or just ditch it.

Reply:
I'd rather just promote the Hindi name without placetype to the longname and ditch this translation. They're strung together from literal placetype translations and most references online just show the name of the region without placetype.